docs/D-Modularity/functions-arrays-and-structs.md

```diff
@@ -80,9 +84,9 @@ The compiler interprets the call to `power()` in the `main()` function as a vali
 
 ### include
 
-We can define a function used by our host application in a file separate from the source file of our application.  If we do, we also store its prototype in a separate file called a header file.  Typically, this header file has the extension .h.  We insert the contents of the header file into the source file of our application using a `#include` directive. 
+We can define a function used by our host application in a file separate from the source file of our application.  If we do, we also store its prototype in a separate file called a header file.  Typically, this header file has the extension `.h`.  We insert the contents of the header file into the source file of our application using a `#include` directive. 
```

Comment:
There are also redundant whitespaces here, "If we do,...", "Typically,..", and "We insert...."

Reply:
Done